src/MeshlessFV/MeshlessFVTree.cpp

```diff
@@ -802,24 +783,21 @@ void MeshlessFVTree<ndim,ParticleType,TreeCell>::UpdateGodunovFluxes
 
     // Add all buffers back to main arrays
     // Could be simply atomic?
-#pragma omp barrier
 #pragma omp critical
     {
       for (i=0; i<Nhydro; i++) {
-	    if (mfvdata[i].flags.check_flag(active))
-	      for (k=0; k<ndim+2; k++) mfvdata[i].dQdt[k] += fluxBuffer[i][k];
-        for (k=0; k<ndim+2; k++) mfvdata[i].dQ[k] += dQBuffer[i][k];
-        for (k=0; k<ndim; k++) mfvdata[i].rdmdt[k] += rdmdtBuffer[i][k];
+        if (mfvdata[i].flags.check_flag(active)) {
```

Comment:
I believe that these change don't actually make a difference. Although we store extra values for dQdt and rdmdt above, we still only use the ones we need here.

I don't really know whether rdmdt is correctly calculated at the moment, I'd have thought that we should build it  up throughout the the timestep. However, I suggest that we forget about it for now - we don't have a working implementation that makes use of it anyway.